recipes/open62541/all/conanfile.py

```diff
@@ -182,6 +194,8 @@ def requirements(self):
             self.requires("libwebsockets/4.3.2")
         if self.options.discovery == "With Multicast" or "multicast" in str(self.options.discovery):
             self.requires("pro-mdnsd/0.8.4")
+        if self.options.nodeset_loader:
+            self.requires("libxml2/2.13.4")
```

Comment:
Libxml2 should be required using version ranges
```suggestion
            self.requires("libxml2/[>=2.12.5 <3]")
```